web/src/main/java/org/fao/geonet/kernel/harvest/harvester/ogcwxs/Harvester.java

```diff
@@ -192,7 +193,7 @@ public OgcWxSResult harvest() throws Exception {
         // Try to load capabilities document
 		this.capabilitiesUrl = getBaseUrl(params.url) +
         		"SERVICE=" + params.ogctype.substring(0,3) +
-        		"&VERSION=" + params.ogctype.substring(3) +
+        		"&ACCEPTVERSIONS=" + params.ogctype.substring(3) +
```

Comment:
If it does not work for CSW or WMS, then they are implementing OGC Web Services Common (OWS) the wrong way. 

Afaik ``version`` was never part of the GetCapabilities request. Conceptually it does also not make sense to ask for a response in a specific version without asking for the supported versions first.